notebooks/04_model_select_and_optimize/nni_surprise_svd.ipynb

```diff
@@ -0,0 +1,835 @@
+{
```

Comment:
<p>Title and initial explanation of nni and this notebook are missing</p>

 _Reply via <a href='https://app.reviewnb.com/Microsoft/Recommenders/pull/687/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Also could you add brief outline of the contents? Maybe simple 1, 2, 3 steps as an overview so that readers can easily see what's yaml file is doing, json is doing, where the compute gets done, what's the prerequisites , etc.